src/data/MapfishPrintProvider.js

```diff
@@ -309,14 +310,22 @@ Ext.define('GeoExt.data.MapfishPrintProvider', {
             });
             store.loadRawData(capabilities);
         } else if (url) { // if servlet url is passed
+            var proxy = {
+                type: 'ajax',
```

Comment:
Maybe better moving setting `type = 'ajax'` to the else part of `if (this.getUseJsonp())`. Technically no difference but it is a cleaner separation since `type: 'ajax'` makes only sense for the non `jsonp` variant.

Reply:
Done in c11352a9d2b9c78835076bf88c41f731ac519395, thanks